course5/homework03/homework03.cc

```diff
@@ -1,18 +1,70 @@
 #include <cstdio>
+#include <string>
+#include <sstream>
 
-// TODO your code goes here
+const int DEFAULT_PRECISION = 0;
+const char* TRUE_STRING = "true";
+const char* FALSE_STRING = "false";
```

Comment:
Bit of an overkill in this case.
Get into habit of marking all namespace-level constants as ```static```. We'll discuss why